Bottom: 57e5712fe046d7e516825252e01b3e89d954c8ea
Top:    f1e2de638128e8eb84bb6a46bdf9f7e05fd3b58f
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-11-24 19:09:45 +0000

Win32 FIX


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index f736c54..f1018b0 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -28,6 +28,10 @@
 #   include <GL/gl.h>
 #endif
 
+#ifdef WIN32
+#   include "hid/common/glext.h"
+#endif
+
 #include <gtk/gtkgl.h>
 #include "hid/common/hidgl.h"
